src/vm/functions/tuples.rs

```diff
@@ -53,3 +54,24 @@ pub fn tuple_get(args: &[SymbolicExpression], env: &mut Environment, context: &L
         _ => Err(Error::new(ErrType::TypeError("TupleType".to_string(), value.clone())))
     }
 }
+
+pub enum TupleDefinitionType {
+    Implicit(Box<[SymbolicExpression]>),
+    Explicit,
+}
+
+// Identify whether a symbolic expression is an implicit tuple structure ((key2 k1) (key2 k2)), 
```

Comment:
Please use `///` for method documentation, not `//`.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1007/commits/ebfc3b6ffae4563b3a544a8aa3b9a609a6e7bbf8